iib/workers/tasks/build.py

```diff
@@ -156,34 +156,40 @@ def _finish_request_post_build(
         index image.
     :param str overwrite_from_index_token: the token used for overwriting the input
         ``from_index`` image.
+    :return: pull spec of the index_image
+    :rtype: str
     :raises IIBError: if the manifest list couldn't be created and pushed
     """
     conf = get_worker_config()
+    state_reason = 'Updated the index image'
     if from_index and overwrite_from_index:
         log.info(f'Ovewriting the index image {from_index} with {output_pull_spec}')
         index_image = from_index
         exc_msg = f'Failed to overwrite the input from_index container image of {index_image}'
         args = [f'docker://{output_pull_spec}', f'docker://{index_image}']
         _skopeo_copy(*args, copy_all=True, dest_token=overwrite_from_index_token, exc_msg=exc_msg)
+        state_reason = "Overwrote the index image {from_index} with {output_pull_spec}"
```

Comment:
Before Matt catches this, could you please be consistent and use single quotes throughout. :)

Reply:
New patch already coming :p

Reply:
And there should be an `f` before the string for this to work `f'Overwrote the index image {from_index} with {output_pull_spec}'`

Reply:
missing in syntax in figuring out the right words already

Reply:
updated